Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -400,10 +554,18 @@ int testModel(std::string data_path, const TFLiteData &tflite_data,
 int main(int argc, char *argv[]) {
   int status = ML_ERROR_NONE;
   if (argc < 3) {
-    std::cout << "./TransferLearning Config.ini resources\n";
-    exit(0);
+#if defined(__TIZEN__)
+    ml_loge("./TransferLearning Config.ini resources.");
+#else
+    std::cout << "./TransferLearning Config.ini resources." << std::endl;
```

Comment:
Could add information or checks about trailing slash in the resource directory?
```suggestion
    std::cout << "./TransferLearning Config.ini resources" << std::endl;
```

Reply:
the trailing slash is handled internally by always adding slash at the end of resources path.

Reply:
This application used to require adding the last slash for the resource path. Check out `meson.build`.  btw. I guess the test argument should be modified as well.

`test('app_training', e, args: [join_paths(res_path, 'Training.ini'), res_path + '/'], timeout: 60)`

Reply:
updated to `test('app_training', e, args: [join_paths(res_path, 'Training.ini'), res_path], timeout: 60)`